iocore/eventsystem/I_Lock.h

```diff
@@ -28,7 +28,7 @@
 #include "ts/Diags.h"
 #include "I_Thread.h"
 
-#define MAX_LOCK_TIME HRTIME_MSECONDS(200)
+static const ts_microseconds MAX_LOCK_TIME(200);
```

Comment:
This could also be
{{code}}
static const ts_nanoseconds MAX_LOCK_TIME(ts_microseconds(200));
{{code}}
That style is used elsewhere, I'm not sure which is better. Denoting this value in nanoseconds avoids conversions later.